### PR TITLE
Dataimport

### PR DESCRIPTION
**Pull request by Evillivekai**

### CODE REVIEW

File: database/database_operate.js [hunk 1]
```diff
@@ -15,6 +15,19 @@ const product_create = (props, callback) => {
   });
 };
 
+const product_readALL = (callback) => {
+  // create an array of product list
+  const queryStr = 'SELECT * FROM products;';
```
Comment (mflinn92) [inline database/database_operate.js]: THis query will take forever and you will overload javascript's heap memory most likely. You probably don't want to ever run this, it'll tank your virtual machine

File: server/index.js [hunk 2]
```diff
@@ -14,42 +14,66 @@ const port = 9000;
 app.use(cors());
 app.use(bodyParser.json());
 app.use(express.static('client/dist'));
+app.use('/:id', express.static('client/dist'));
```
Comment (mflinn92) [inline server/index.js]: I don't think you need /:id here. Static is for requests to the root url normally

File: database/database_operate.js (hunk 1 above)
Comment (Evillivekai) [reply database/database_operate.js]: You are right, I might need to desc maybe only top 50.

File: server/index.js [hunk 3]
```diff
@@ -14,42 +14,66 @@ const port = 9000;
...
+//       return;
+//     }
+//     // eslint-disable-next-line no-console
+//     console.log('ikea_products connected');
+//     res.send(data).status(200);
+//   });
+// });
+
+app.get('/api/products/:id', (req, res) => {
+  dbOperate.product_readALL((err, data) => {
```
Comment (mflinn92) [inline server/index.js]: Same comment as before. Don't retrieve every product from db, it will cause problems

Comment (Evillivekai) [reply server/index.js]: Got it, I will fix them.

**Pull request merged**